Lib/logging/config.py

```diff
@@ -60,15 +61,24 @@ def fileConfig(fname, defaults=None, disable_existing_loggers=True, encoding=Non
     """
     import configparser
 
+    if isinstance(fname, str):
+        if not os.path.exists(fname):
+            raise FileNotFoundError(f"{fname} doesn't exist")
+        elif not os.path.getsize(fname):
+            raise ValueError(f'{fname} is an empty file')
```

Comment:
ValueError is generally only used when the value itself is a problem, not when something external to the process such as a filesystem doesn't look right.  Same below.  If in doubt as to what exception to use here, I'd just go with RuntimeError if you don't have an Error exception class for your package already.